src/new/wallet.js

```diff
@@ -296,19 +297,13 @@ class HathorWallet extends EventEmitter {
       let promise;
       if (this.firstConnection) {
         this.firstConnection = false;
-        promise = wallet.loadAddressHistory(
-          0,
-          wallet.getGapLimit(),
-          this.conn,
-          this.store,
-          this.preCalculatedAddresses
-        );
+        promise = this.loadAddresses(0, wallet.getGapLimit());
```

Comment:
Does it make sense (in the future) for the gap limit to be configured for each wallet? In that case, we would have a new parameter to set the gap limit for this wallet.